aries-site/src/pages/components/layer/index.mdx

```diff
@@ -0,0 +1,157 @@
+| **2**  | **Body**     | Any additional content.                                | optional  | --                                                                         |
+| **3**  | **Footer**   | Contains layer actions.                                | optional  | The ordering and alignment of footer actions depends on the type of layer. |
+
+<StyleTable
+  data={[
+    {
+      label: 4,
+      property: 'pad',
+      value: 'medium',
+    },
+    {
+      label: 5,
+      property: 'gap',
+      value: 'medium',
+    },
+    {
+      label: 6,
+      property: 'round',
+      value: 'small',
+    },
+    {
+      label: 7,
+      property: 'elevation',
+      value: 'large',
+    },
+  ]}
+/>
+
+## Types of layers
+
+Different use cases and content warrant different layer types. Learn more about each type below.
+
+<LayerTypes />
+
+## Layer closing behavior
+
+The closing behavior and anatomy of a layer will differ depending on if the layer is informational or actionable.
+
+### Informational
+
+For informational layers that do not require any user actions, place a close button in the layer header.
+
+<Example plain>
+  <InformationalLayerClose />
+</Example>
+
+- The "X" button should close the layer.
+- A user may also click outside of the layer or use the Escape key to close it. To implement, use `onClickOutside` and `onEsc` on Layer.
+
+### Actionable
+
+For actionable layers that require user actions, place a "Cancel" action alongside the other footer actions. In some cases, a label other than "Cancel" may be appropriate.
+
+Do not include an "X" button in the header to avoid confusion of how the user should close the layer.
+
+<Example plain>
+  <ActionableLayerClose />
+</Example>
+
+- The "Cancel" button should reject any changes and close the layer.
+  To prevent the user from losing progress due to accidental clicks outside of a layer or Escape key presses, `onEsc` and `onClickOutside` functionality should not be enabled.
+
+<Notification
+  status="info"
+  message="The ordering and alignment of footer actions depends on the type of layer. See anatomy diagrams for specific layer types for details."
+  actions={[
+    {
+      label: 'See layer types',
+      icon: <FormNext />,
+      href: '#types-of-layers',
+      reverse: true,
+    },
+  ]}
+  width="large"
+  margin={{ bottom: 'medium' }}
+/>
+
+## Action label conventions
+
+If a layer contains footer actions, the primary action should be labeled using verb + noun format (e.g., "Apply filters").
+
+The noun may be omitted from actions labels beyond the primary action if the action is clear and unambiguous (e.g. "Reset" instead of "Reset filters").
+
+When in doubt, follow the verb + noun convention to ensure clarity.
+
+## Overflow behavior
+
+If a layer contains filters, the footer actions should be persistent and sticky at the bottom of the layer.
```

Comment:
This might benefit from a sentence on "why" we provide this guidance and why it is different from general form contexts.

For instance, we say this for the Form context:
> to encourage the user to see the entire form before submission.

which is a helpful "why".

Reply:
Added a "why"